Added blog posts (#363)
gardener/documentation: website/blog/2022-08/00/Gardener-Community-Meeting-July-News.md

---
title: Upcoming Community Call - Cilium / Isovalent Presentation
linkTitle: Upcoming Community Call - Cilium / Isovalent Presentation
newsSubtitle: October 6, 2022
publishdate: 2022-08-10
archivedate: 2022-10-07
authors:
- name: Nikolay Boshnakov
  email: nikolay.boshnakov@sap.com
  avatar: https://avatars.githubusercontent.com/u/25197046?s=400&u=56175926393a77892662001f0dca5a439d1e771f&v=4
aliases: ["/blog/2022/08/10/01"]
---

## Presenters

This community call will be led by [Raymond de Jong](https://github.com/raymonddejong).

## Topics

This meeting will explore the uses of [Cilium](https://cilium.io/), an open source software used to secure the network connectivity between application services deployed using Kubernetes, and [Hubble](https://github.com/cilium/hubble/blob/master/README.md), the networking and security observability platform built on top of it.

Viewers will be shown how to check for connectivity and how to carry out basic troubleshooting. Later on in the meeting, an overview will be given of the network policy and the procedure for switching from Calico will be demonstrated.

> There is going to be time after the main part to answer questions and look at any issues you might have encountered. Come join us!

Reviewer: ```suggestion
Q&A is planned after the main part to look at any issues you might have encountered. Come join us!
```
